index.html

```diff
@@ -3,13 +3,25 @@
 		<title> Westie World </title>
 		<link rel="stylesheet" type="text/css" href="stylesheet.css" />
 		<link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Open+Sans" />
+		<!-- <link rel="stylesheet" type="image" href="public/westieWorld.png"/> -->
```

Comment:
Do we still need these lines?

Reply:
Nope. Cleaning these up as we speak. Although, I might run and get some food first lol